sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-notification/e-cert-notification.ts

```diff
@@ -0,0 +1,70 @@
+import { EntityManager } from "typeorm";
+import { EligibleECertDisbursement } from "../disbursement-schedule.models";
+import { ProcessSummary } from "@sims/utilities/logger";
+
+/**
+ * Provides a basic structure for creating and
+ * sending notifications related to e-Cert disbursements.
+ */
+export abstract class ECertNotification {
```

Comment:
One minor comment to mention. As this is a base class to create notifications, the student data is a core common part that can be built from a base class method. And that base class method can be called inside `createNotification()`

Reply:
The idea was that each notification could evolve differently and the data required for each should be handled in the inherited class. For instance, a notification for the Ministry could potentially contain further information about the blocked disbursement or something else. Assuming all notifications will have the student data as a core would not be accurate IMO.

Reply:
Excluding ministry emails for institutions, other ministry notifications have student first and last name as common. But not a blocker considering the present.